WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.m

```diff
@@ -317,6 +317,9 @@ - (void)showDetailsForNoteWithID:(NSString *)notificationID
     if (notification) {
         DDLogInfo(@"Pushing Notification Details for: [%@]", notificationID);
         
+        NSDictionary *properties = @{ @"type" : (notification.type ?: [NSString string]) };
+        [WPAnalytics track:WPAnalyticsStatPushNotificationAlertPressed withProperties:properties];
```

Comment:
Is it weird to pass a dictionary with an empty string for a type? Why not just pass nil for properties?